src/aosm/azext_aosm/definition_folder/reader/bicep_definition.py

```diff
@@ -153,6 +159,72 @@ def _artifact_manifests_exist(
 
         return ManifestsExist.NONE
 
+    @staticmethod
+    def _base_Resources_exist(
+        config: BaseCommonParametersConfig, command_context: CommandContext
+    ) -> bool:
+        """
+
+        Returns True if all required manifests exist, False if none do, and raises an

```

Comment:
Think this is a typo, from copying over the docstring from the check manifest exists, should be return true if all base resourcs exist

Reply:
Yeah, forgot to change it. I have fixed it now.